test/e2e/test/logstash/checks.go

```diff
@@ -176,7 +176,12 @@ func (b Builder) CheckStackTestSteps(k *test.K8sClient) test.StepList {
 				Password: password,
 			},
 			Want{
-				Match: map[string]string{"status": "green"},
+				MatchFunc: map[string]func(string) bool{
+					// only red is considered as failure in health API
+					"status": func(status string) bool {
+						return status != "red"
+					},
```

Comment:
Is there not a chance that we miss another value, like an empty string for example? I was wondering if we could explicitly expect the `green` or `yellow` values, and reuse a function.
```suggestion
						"status": isGreenOrYellow,
```

With `isGreenOrYellow` being:

```golang
// isGreenOrYellow returns true if the status is either green or yellow, red is considered as failure in health API.
func isGreenOrYellow(status string) bool {
	return status == "green" || status == "yellow"
}
```

Reply:
The status only gives one of the color, no empty string